samtranslator/plugins/api/implicit_api_plugin.py

```diff
@@ -195,6 +204,32 @@ def _add_api_to_swagger(self, event_id, event_properties, template):
         resource.properties["DefinitionBody"] = editor.swagger
         template.set(api_id, resource)
 
+    def _maybe_add_condition_to_implicit_api(self, template_dict):
+        """
+        Decides whether to add a condition to the implicit api resource.
+        :param dict template_dict: SAM template dictionary
+        """
+        if None not in self.conditions and len(self.conditions) > 0:
```

Comment:
That's for functions that don't have conditions on them. None still gets added to this conditions list/set.

Reply:
If there is a function that references the implicit API that doesn't have a condition on it, then SAM doesn't need to create an aggregate condition for that API (it will always need to create the API).